apps/hash-ai-worker-ts/src/activities/shared/get-llm-response.ts

```diff
@@ -164,7 +164,9 @@ const sanitizeInputBeforeValidation = (params: {
   return input;
 };
 
-const ajv = new Ajv();
+const ajv = new Ajv({
+  strict: false,
+});
```

Comment:
What is the exact failure here? We can probably make the configuration more precise to avoid failures we don't want to allow creeping in

If it's to do with us using non-standard JSON Schema keywords, we can [add them](https://ajv.js.org/strict-mode.html#prohibit-ignored-keywords)

Reply:
I set this because of the `label` key is used in the graphics card data types, and it's not a supported JSON schema keyword. I've manually added the JSON schema keyword and removed `strict: false`, and tested it to see if it works in https://github.com/hashintel/hash/pull/4505/commits/b4cf7d7e422ac3a7eea6c9ea4b6220cd8b9a850f

I've created a [follow-up ticket](https://linear.app/hash/issue/H-2840/filter-out-label-property-from-dereferenced-entity-types) for filtering out the `label` property from the generated JSON schemas when passing them to the LLM. The reason I didn't directly remove it in the `dereferenceEntityType` method, is because I thought the `label` might be useful for the LLM to better understand the data type.

Reply:
We can just leave it in as additional context, and remove it if it becomes a problem